react/features/speaker-stats/functions.js

```diff
@@ -63,56 +62,66 @@ export function getPendingReorder(state: Object) {
 }
 
 /**
- * Get sorted speaker stats based on a configuration setting.
+ * Get sorted speaker stats ids based on a configuration setting.
  *
  * @param {Object} state - The redux state.
  * @param {Object} stats - The current speaker stats.
- * @returns {Object} - Ordered speaker stats.
+ * @returns {Object} - Ordered speaker stats ids.
  * @public
  */
-export function getSortedSpeakerStats(state: Object, stats: Object) {
+export function getSortedSpeakerStatsIds(state: Object, stats: Object) {
     const orderConfig = getSpeakerStatsOrder(state);
 
     if (orderConfig) {
         const enhancedStats = getEnhancedStatsForOrdering(state, stats, orderConfig);
-        const sortedStats = objectSort(enhancedStats, (currentParticipant, nextParticipant) => {
-            if (orderConfig.includes('hasLeft')) {
-                if (nextParticipant.hasLeft() && !currentParticipant.hasLeft()) {
-                    return -1;
-                } else if (currentParticipant.hasLeft() && !nextParticipant.hasLeft()) {
-                    return 1;
-                }
-            }
 
-            let result;
-
-            for (const sortCriteria of orderConfig) {
-                switch (sortCriteria) {
-                case 'role':
-                    if (!nextParticipant.isModerator && currentParticipant.isModerator) {
-                        result = -1;
-                    } else if (!currentParticipant.isModerator && nextParticipant.isModerator) {
-                        result = 1;
-                    } else {
-                        result = 0;
-                    }
-                    break;
-                case 'name':
-                    result = (currentParticipant.displayName || '').localeCompare(
-                        nextParticipant.displayName || ''
-                    );
-                    break;
-                }
+        return Object.entries(enhancedStats).sort(([ , a ], [ , b ]) => compareFn(a, b))
+.map(el => el[0]);
```

Comment:
maybe fix the indentation here
```suggestion
Object.entries(enhancedStats)
        .sort(([ , a ], [ , b ]) => compareFn(a, b))
        .map(el => el[0]);
```